framework/src/test/java/io/github/kgress/scaffold/webelement/AbstractWebElementTests.java

```diff
@@ -99,7 +100,7 @@ public void testGetTagName() {
     public void testIsEnabled() {
         var testElement = new TestableAbstractWebElement(mockElement1);
 
-        assertTrue("The element should be enabled by default", testElement.isEnabled());
+        assertTrue(testElement.isEnabled(), "The element should be enabled by default");
```

Comment:
....something isn't right here. i know that different assert packages take different orders for parameters... and the fact that this one takes the value first, but the assertFalse on line 114 takes the comment string first. if anything, please make these consistent (plus, this will ensure you don't accidentally call the wrong assertTrue() method)

Reply:
woah, good catch. I caught the assertTrue's but didn't notice the others. Will update that.